weatherman.py

```diff
@@ -146,6 +146,22 @@ def avg_mean_humid(self, by = None):
         else:
             return None
 
+
+    def draw_chart_for_month(self, by = None):
```

Comment:
Read through https://www.python.org/dev/peps/pep-0008/? 
This is code quality standards that we follow. According to this every method should have docstring and here you should not put any spaces around = when use in method. It should be something like
```
def draw_chart_for_month(self, by=None):
```

Reply:
Also class should be inherited from object class of python like following 
```
class WeathermanEntries(object):
```